pulp_ansible/app/galaxy/v3/serializers.py

```diff
@@ -48,6 +49,14 @@ def get_href(self, obj):
             kwargs={"path": self.context["path"], "namespace": obj.namespace, "name": obj.name},
         )
 
+    def get_deprecated(self, obj):
+        """Get href."""
+        distro = get_object_or_404(models.AnsibleDistribution, base_path=self.context["path"])
+        deprecation = distro.repository.cast().collection_memberships.filter(
+            collection_id=obj.collection.pk
+        )
+        return bool(deprecation.count())
```

Comment:
I believe `exists()` https://docs.djangoproject.com/en/3.1/ref/models/querysets/#django.db.models.query.QuerySet.exists would be more efficient.